dev/gulpfile.js

```diff
@@ -1,2 +1,50 @@
 /* eslint-disable import/no-extraneous-dependencies */
-require('require-dir')('./gulp-tasks');
+
+const path = require('path');
+
+const gulp = require('gulp');
+const sourcemaps = require('gulp-sourcemaps');
+
+const postcss = require('gulp-postcss');
+const autoprefixer = require('autoprefixer');
+const pxtorem = require('postcss-pxtorem');
+
+const STATIC = path.resolve(__dirname, '..', 'static');
+const CSS = path.resolve(STATIC, 'target', 'stylesheets');
+const JS = path.resolve(STATIC, 'src', 'javascripts-legacy');
+const TRANSPILED = path.resolve(STATIC, 'transpiled', 'javascripts');
```

Comment:
hmmm, is that a default setting in intellij then? it's ignore by git, does intellij not observe that?